modules/oe_theme_helper/oe_theme_helper.services.yml

```diff
@@ -36,6 +36,10 @@ services:
     arguments: ['@current_route_match', '@entity_type.manager', '@entity.repository']
     tags:
       - { name: event_subscriber }
+  oe_theme_helper.icon_options_subscriber:
+    class: Drupal\oe_theme_helper\EventSubscriber\IconOptionsEventSubscriber
+    tags:
+      - { name: event_subscriber }
```

Comment:
Sorry, but I didn't get why event subscriber is implemented on oe_theme side again. As I see it is already done in oe_paragraph. Should it be updated only in oe_paragraph?

Reply:
I replied in the ticket.

Reply:
Thanks. Looks good to me.